src/core/src/package_managers/YumPackageManager.py

```diff
@@ -325,26 +325,126 @@ def extract_dependencies(self, output, packages):
+        # =============================================================================================
+        # Install  2 Packages
+        #
+        # Total download size: 72 M
+        # Installed size: 93 M
+        # Operation aborted.
+
+        # In Yum 4: Sample 3 (Both upgrades and installations required):
+        # Sample output for the cmd 'sudo yum update --assumeno kernel-modules.x86_64' is :
+        #
+        # Last metadata expiration check: 0:01:56 ago on Tue 25 Jul 2023 12:01:47 PM UTC.
+        # Dependencies resolved.
+        # ================================================================================================
+        # Package             Arch    Version               Repository                               Size
+        # ================================================================================================
+        # Upgrading:
+        # kernel-tools        x86_64  4.18.0-372.64.1.el8_6 rhel-8-for-x86_64-baseos-eus-rhui-rpms  8.4 M
+        # kernel-tools-libs   x86_64  4.18.0-372.64.1.el8_6 rhel-8-for-x86_64-baseos-eus-rhui-rpms  8.2 M
+        # openssl             x86_64  1:1.1.1k-9.el8_6      rhel-8-for-x86_64-baseos-eus-rhui-rpms  710 k
+        # openssl-libs        x86_64  1:1.1.1k-9.el8_6      rhel-8-for-x86_64-baseos-eus-rhui-rpms  1.5 M
+        # Installing dependencies:
+        # kernel-core         x86_64  4.18.0-372.64.1.el8_6 rhel-8-for-x86_64-baseos-eus-rhui-rpms   40 M
+        # kernel-modules      x86_64  4.18.0-372.64.1.el8_6 rhel-8-for-x86_64-baseos-eus-rhui-rpms   32 M
+        #
+        # Transaction Summary
+        # ================================================================================================
+        # Install  2 Packages
+        # Upgrade  4 Packages
+
+        # In Yum 4: Sample 4 (dependent patch detail split over 2 lines):
+        # Sample output for the cmd 'sudo yum update --assumeno polkit.x86_64' is :
+        #
+        # Last metadata expiration check: 0:08:47 ago on Tue 25 Jul 2023 02:14:28 PM UTC.
+        # Package polkit-0.115-13.el8_5.2.x86_64 is already installed.
+        # Dependencies resolved.
+        # ================================================================================
+        # Package   Arch   Version          Repository                              Size
+        # ================================================================================
+        # Upgrading:
+        # polkit    x86_64 0.115-14.el8_6.1 rhel-8-for-x86_64-baseos-eus-rhui-rpms 154 k
+        # polkit-libs
+        #           x86_64 0.115-14.el8_6.1 rhel-8-for-x86_64-baseos-eus-rhui-rpms  77 k
+        #
+        # Transaction Summary
+        # ================================================================================
+        # Upgrade  2 Packages
+        #
+        # Total download size: 231 k
+        # Operation aborted.
+
         dependencies = []
+        package_arch_to_look_for = ["x86_64", "noarch", "i686"]
```

Comment:
You can close this -- but if you can find the arm64 string, it's good to include that in one shot. Not a blocker to this PR.

Reply:
This may also be initialized once as a constant in the constructor of the class.

Reply:
Please add "aarch64" to the list example package: [**libbroker-6.0.4-1.el9.aarch64.rpm**](https://rhel.pkgs.org/9/epel-testing-aarch64/libbroker-6.0.4-1.el9.aarch64.rpm.html)

Reply:
Done